engine/withdraw.go

```diff
@@ -0,0 +1,90 @@
+package engine
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/gofrs/uuid"
+	withdrawal "github.com/thrasher-corp/gocryptotrader/database/repository/withdraw"
```

Comment:
withdrawal clashes a little bit, is it possible to change this to db? Or Add a StoreEvent func in management/withdraw package?

Reply:
Happy to rename it to something else
adding a helper to portfolio/withdraw package will cause all kinds of import cycle issues without a major restructure